components/entrySignUp/entrySignUp.js

```diff
@@ -80,11 +80,20 @@ Template.entrySignUp.helpers({
       return "border: 1px solid #a94442";
     } else if (ActiveEntry.errorMessages.equals('fullName', "Name is probably not complete")) {
       return "border: 1px solid #f2dede";
-    } else if (ActiveEntry.errorMessages.equals('fullName', "Name present")) {
+    } else if (ActiveEntry.successMessages.equals('fullName', "Name present")) {
       return "border: 1px solid green";
     } else {
       return "border: 1px solid gray";
     }
+  },
+  signUpErrorMessages: function() {
+    var errorMessages = [];
+    Object.keys(ActiveEntry.errorMessages.all()).forEach(function(key) {
+      if (key !== "signInError" && ActiveEntry.errorMessages.get(key)) {
+        errorMessages.push(ActiveEntry.errorMessages.get(key));
+      }
+    });
+    return errorMessages;
```

Comment:
Actually I wanted to show all errors at a time, but it was just my decision. We can show the first error message at a time with something like the following code. Would you like me to do it for simplicity?
`
var allErrorMessages = Object.keys(ActiveEntry.errorMessages.all()).filter(function(key) {
  return key !== "signInError" && ActiveEntry.errorMessages.get(key);
});
return ActiveEntry.errorMessages.get(allErrorMessages[0]);
`
Also I will open an issue to discuss successMessages.